### PR TITLE
fix issue with linting of YAML files

### PR DESCRIPTION
**Pull request by hansinikarunarathne**
# Pull Request Template for Kubeflow manifests Issues

## ✅ Contributor checklist
  - Make sure you have tested with kustomize. See [Installation Prerequisites](https://github.com/kubeflow/manifests#prerequisites)
  - All the commits have been _signed-off_  (To pass the `DCO` check)
  - Submit the [Contributor License Agreements](https://cla.developers.google.com/clas) (To pass the `cla/google` check)

---     
 
> You can join the CNCF Slack and access our meetings at the [Kubeflow Community](https://www.kubeflow.org/docs/about/community/) website. Our channel on the CNCF Slack is here [**#kubeflow-platform**](https://app.slack.com/client/T08PSQ7BQ/C073W572LA2).

### CODE REVIEW

**Comment by hansinikarunarathne** [comment]
@juliusvonkohout I fix the mistake. Before merge #2815, I think we need to merge this to master and sync this change with that PR.

File: .github/workflows/linting_bash_python_yaml_files.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
     - name: Set up changed files
       id: changed_files
       run: |
-        git diff --name-status origin/master...HEAD | grep -E '^[AM].*\.(yaml|yml)$' | awk '{print $2}' > changed_files_in_PR.txt || true
+        git diff --name-status origin/master...HEAD | grep -E '^[AM].*\/(common|example)\/.*\.(yaml|yml)$' | awk '{print $2}' > changed_files_in_PR.txt || true
```
Comment (juliusvonkohout) [inline .github/workflows/linting_bash_python_yaml_files.yaml:74]: can you explain a bit? doe sthis only check /common/* and /example/* for now?

Comment (hansinikarunarathne) [reply .github/workflows/linting_bash_python_yaml_files.yaml:74]: Yes. as I remember Before we discussed doing that. That's why I added It. Any concerns regarding that ?

Comment (juliusvonkohout) [reply .github/workflows/linting_bash_python_yaml_files.yaml:74]: Yes, please add .github and tests and hack next to common and example

Comment (juliusvonkohout) [reply .github/workflows/linting_bash_python_yaml_files.yaml:74]: but please fix this in a follow up PR.

Comment (hansinikarunarathne) [reply .github/workflows/linting_bash_python_yaml_files.yaml]: Add those files as well

**Comment by juliusvonkohout** [comment]
This is not working properly. I see
```
Run cat changed_files_in_PR.txt
  cat changed_files_in_PR.txt
  shell: /usr/bin/bash -e {0}
No YAML files have changed in this PR.
```

but you clearly changed yaml files here.

**Comment by hansinikarunarathne** [comment]
> This is not working properly. I see
> 
> ```
> Run cat changed_files_in_PR.txt
>   cat changed_files_in_PR.txt
>   shell: /usr/bin/bash -e {0}
> No YAML files have changed in this PR.
> ```
> 
> but you clearly changed yaml files here.

fixed the issue

**Pull request merged**